src/runtime/msg/action.rs

```diff
@@ -34,6 +34,7 @@ use crate::{
 pub enum ActionCtx {
     Authenticate(AuthRequest),
     Logout,
+    DeleteAccount(String),
```

Comment:
for fields that don't have a type, I'd suggest to add a field instead of just the type:
`DeleteAccount { password: String }`

Reply:
I can create a type for Password which would just be an alias for String if you want?
Using a struct for this is not very useful as we don't have other arguments to send and result in more cumbersome API